README.md

```diff
@@ -54,81 +62,100 @@ The architecture is composed of two layers:
+1. Clone the repository (including the submodules using: `--recurse-submodules`)
 
-# Setup the build envirnoment
+    ```
+    git clone --recurse-submodules https://github.com/mit-dci/opencbdc-tx
+    ```
 
-Use these directions if you want to build the source on your machine.
-If you just want to run the system, see "Run the Code" below.
+## Setup the build environment
+
+Use these directions if you would like to build the source code on your machine.
+Alternatively, if you just want to run the system, skip to the [Run the Code](#run-the-code) section below.
+
+1. Setup the build-environment
 
-1. Setup the build-environment.
    Note that this script is just a convenience to install system-wide dependencies we expect.
    As a result, it uses the system package manager, requires `sudo`, and should only be run **once**.
    ```console
    # ./scripts/install-build-tools.sh
    ```
 1. Setup project dependencies
-   This script builds and installs a local copy of several build-dependencies which are not widely packaged.
-   Because it installs to a local, configurable prefix (defaulting to `./prefix`), it does not need root permissions to run.
-   Furthermore, it should always be safe to delete `prefix` and rerun this script.
+
+   This script builds and installs a local copy of several build-dependencies that are not widely packaged.
+   Because the installation is done in a user-specific location (`./prefix` by default) rather than a system-wide one, you do not need administrative (root) permission to run the script. Additionally, if you want to remove the installed build-dependencies or restart the installation process, you can safely delete the `prefix` directory and rerun the script.
    ```console
    $ ./scripts/setup-dependencies.sh
    ```
-1. Run the build
+1. Run the build script
    ```console
    $ ./scripts/build.sh
    ```
 
-## macOS
-Note that if you have not already installed the xcode cli tools you will need to:
+### macOS
+Note: If you have not already installed the Xcode CLI tools, you will need to do so:
 
 ```console
 # xcode-select --install
 ```
-# Run the Code
+## Run the Code
 
 The API Reference is now housed in [an external repository](https://github.com/mit-dci/opencbdc-tx-pages/).
-See the [live deployment](https://mit-dci.github.io/opencbdc-tx-pages/) to browse.
+See the [Documentation](#documentation) section below to browse.
 
-## UHS-based Architectures (2PC & Atomizer)
+### UHS-based Architectures (2PC & Atomizer)
 See the [2PC & Atomizer User Guide](docs/2pc_atomizer_user_guide.md)
-## PArSEC Architecture
+### PArSEC Architecture
 See the [PArSEC User Guide](docs/parsec_user_guide.md)
+
+# Documentation
+
+The official documentation for OpenCBDC can be found here: [OpenCBDC-tx-pages](https://mit-dci.github.io/opencbdc-tx-pages/)
+
 # Testing
 
-Running Unit & Integration Tests
+Users can run both unit/integration and end-to-end tests on OpenCBDC.
```

Comment:
nit: I would be inclined to say something like "verify setup by running tests on OpenCBDC" instead of "Users can run ..."